src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private final String firstName = "apple";
+    private final int firstAmount = 5;
+    private final String secondName = "banana";
+    private final int secondAmount = 10;
+    private final String thirdName = "pineapple";
+    private final int thirdAmount = 15;
+    private final FruitDao fruitDao = new FruitDaoImpl();
+
+    @Test
+    public void addFruit_Ok() {
+        Storage.fruits.clear();
```

Comment:
you use this line in all test methods, move it to the method with annotation @AfterEach

Reply:
:)